app/models/xml/manifest.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module XML
```

Comment:
It would be better to have separate tests for this since it's a separate file.

Reply:
k, I'll see about splitting out these tests from the package_spec